### ISSUE TITLE
one biginner's problem 

### ISSUE THREAD

**Issue by hs805430** [csyben/PYRO-NN-Layers#7]
Excuse me, I just download the  pyro-nn-layers code from this github , but I see a lot of  ‘.cc ’ files in it. Do I need to run through these files first? I tried to run the related '.py' files in  'examples' in pyronn-master folder directly in PyCharm, but there are many errors.

**Comment by csyben** [comment]
I am not sure how to respond to this question. Maybe read the readme first. In short, with this repository the operators are built wrapped in a Python package.

**Comment by hs805430** [comment]
> I am not sure how to respond to this question. Maybe read the readme first. In short, with this repository the operators are built wrapped in a Python package.

Dear author, thanks for your reply.After preliminary code learning, I have been able to run the code file you shared normally and get the same result. I have to say that your work is very valuable. Since I am a novice, I encountered a problem when running your code with my own data set - I tried to use the 3D projection data of CBCT collected by our laboratory equipment as training data, and the results reconstructed by FDK algorithm as learning tags, and made some minor modifications to your original code, but the running results were very discontent. The training loss is about 680000.  My own understanding is that due to the limited performance of the graphics process unit, I set up the batch_ size should not be more than 5, otherwise get the error of the GPU resource exhausted, which makes the model difficult to converge; On the other hand, my proj. data hold the dimension size of (360,512,384), but the result of FDK is (100,512,512)......

Another question is whether I need to change the ramp filter used in the example of binary reconstruction you gave to ram_ 3D？

Here is the code file I tried to change. If you could spare a few minutes to analyze the problem for me, I would be very gratefu！！！

**Comment by hs805430** [comment]
[PYRO-NN-master.zip](https://github.com/csyben/PYRO-NN-Layers/files/6179995/PYRO-NN-master.zip)
[PYRO-NN-Layers-master.zip](https://github.com/csyben/PYRO-NN-Layers/files/6179996/PYRO-NN-Layers-master.zip)
